src/main/scala/no/ndla/learningpathapi/service/ConverterServiceComponent.scala

```diff
@@ -21,7 +22,7 @@ trait ConverterServiceComponent {
 
   class ConverterService {
     def asEmbedUrl(embedUrl: api.EmbedUrl): domain.EmbedUrl = {
-      domain.EmbedUrl(embedUrl.url, embedUrl.language, embedUrl.embedType)
+      domain.EmbedUrl(embedUrl.url, embedUrl.language, EmbedType.valueOfOrDefault(embedUrl.embedType))
```

Comment:
Trenger vi noe form for validering på `embedType`? Det kunne kanskje vært kjekt å returnere en feilmelding til brukeren dersom han prøver å legge til et steg med en `embedType` som ikke er "oembed" eller "lti"